stdlib/public/core/Sequence.swift

```diff
@@ -1204,15 +1228,29 @@ extension Sequence where
     return AnySequence(result)
   }
   
-  /// Returns a subsequence by skipping elements while `predicate` returns
-  /// `true` and returning the remaining elements.
+  /// Returns a subsequence starting after the initial, consecutive elements
+  /// that satisfy the given predicate.
```

Comment:
I feel like the original "skipping" wording was clearer regarding the possible consuming nature of this (a subsequence starting after" implies this is a "view" onto some underlying elements)